.github/PULL_REQUEST_TEMPLATE/ttnn.md

```diff
@@ -0,0 +1,17 @@
+## Related Issues
+
+Please link related issues and describe the problem this pull-request is trying to resolve.
+
+## Description
+
+Please include a summary of the changes and the related issues. This template is for changes in the `ttnn` subdirectory.
+- Describe *what* changed in the pull-request
+- Describe *why* these changes were introduced
+- Describe implications of these code-changes: e.g. runtime/compile-time perf impact, usability, future code-changes
+
+## Checklist
+
+- [ ] Post commit CI passes.
+- [ ] Related model testing
+- [ ] New tests added
+- [ ] Change is made to ttnn and not tt_lib :)
```

Comment:
I have mixed feelings about auto generated lists.   When they do not apply you have to explicitly delete them.   That is good when you want to drive behavior towards best practices.  (Maybe this is a good way...)     Let's talk more about this as well.

Reply:
Discussed in our team-sync. I think that's the right way to look at it; this should serve as reminders to help drive developers towards best practices.